packages/@uppy/dashboard/src/components/FileItem/FileInfo/index.js

```diff
@@ -3,49 +3,67 @@ const prettierBytes = require('@transloadit/prettier-bytes')
 const truncateString = require('@uppy/utils/lib/truncateString')
 
 const renderFileName = (props) => {
-  // Take up at most 2 lines on any screen
-  let maxNameLength
-  // For very small mobile screens
-  if (props.containerWidth <= 352) {
-    maxNameLength = 35
-  // For regular mobile screens
-  } else if (props.containerWidth <= 576) {
-    maxNameLength = 60
-  // For desktops
-  } else {
-    maxNameLength = 30
+  const { author, name } = props.file.meta
+
+  function getMaxNameLength () {
+    if (props.containerWidth <= 352) {
+      return 35
+    }
+    if (props.containerWidth <= 576) {
+      return 60
+    }
+    // When `author` is present, we want to make sure
+    // the file name fits on one line so we can place
+    // the author on the second line.
+    return author ? 20 : 30
   }
 
   return (
-    <div className="uppy-Dashboard-Item-name" title={props.file.meta.name}>
-      {truncateString(props.file.meta.name, maxNameLength)}
+    <div className="uppy-Dashboard-Item-name" title={name}>
+      {truncateString(name, getMaxNameLength())}
     </div>
   )
 }
 
-const renderFileSize = (props) => (
-  props.file.size
-    && (
-    <div className="uppy-Dashboard-Item-statusSize">
-      {prettierBytes(props.file.size)}
+const renderAuthor = (props) => {
+  const { author } = props.file.meta
+
+  if (!author) {
+    return null
+  }
+
+  return (
+    <div className="uppy-Dashboard-Item-author">
+      {props.i18n('by')}
+      {' '}
+      <a
+        href={`${author.url}?utm_source=Companion&utm_medium=referral`}
+        target="_blank"
+        rel="noreferrer"
```

Comment:
Specifying noopener is still necessary for older browsers, but feel free to disregard as we officially dropped support for those.
```suggestion
        rel="noopener noreferrer"
```